Cyuubi/LemonLime
# [WIP] Improved CPU bus implementation

TODO:

- Test absolutely everything

- Make sure everything is endian-independent (not completely sure how C# handles those cases)

- Move ITCM/DTCM to the ARM9 core and the MPCore PMR to the ARM11 core

- Add WriteUInt16/WriteUInt8 operations that don't rely on RMW

- Better memory mapped peripheral support (parent bus, update on tick, etc)

- Anything else I forgot about
